src/fixtures/PlaceCodeProcessor.ts

```diff
@@ -0,0 +1,14 @@
+import { IProcessor } from 'typeorm-fixtures-cli';
+import { PlaceCode } from '../entities/euring-codes/place-code-entity';
+
+/* eslint-disable class-methods-use-this */
+export default class PlaceCodeProcessor implements IProcessor<PlaceCode> {
+  public preProcess(_name: string, object: any): any {
+    console.log(JSON.stringify(object, null, 2));
```

Comment:
remove log and also confirm that result of dummy feeling are available on `pre` stage of processing